packages/core/src/controllers/SendController.ts

```diff
@@ -91,7 +91,7 @@ export const SendController = {
 
   sendToken() {
     if (this.state.token?.address && this.state.sendTokenAmount && this.state.receiverAddress) {
-      SendController.sendERC20Token({
+      this.sendERC20Token({
```

Comment:
If we are not calling the `sendToken` from a different context (like passing it to another object's callback etc.) this shouldn't make any difference, curious what was the issue here 🤔

Reply:
There was no issue here, but I think this is more clean.